tools/RAiDER/models/ecmwf.py

```diff
@@ -321,39 +321,24 @@ def _load_pressure_level(self, filename, *args, **kwargs):
         self._t = t
         self._q = q
 
-        geo_hgt = z / self._g0
+        geo_hgt = (z / self._g0).transpose(1, 2, 0)
 
         # re-assign lons, lats to match heights
         self._lons, self._lats = np.meshgrid(lons, lats)
 
         # correct heights for latitude
-        self._get_heights(_lats, geo_hgt)
+        self._get_heights(self._lats, geo_hgt)
 
-        self._p = np.broadcast_to(levels[:, np.newaxis, np.newaxis],
+        self._p = np.broadcast_to(levels[np.newaxis, np.newaxis, :],
```

Comment:
It was broken before in numerous places actually because variables had been renamed and shapes realigned. I shouldve issued a separate PR but had to get it working to compare with the HRRR.

It results in sub-mm differences with ERA5 model levels.